src/tables/sortable-table.js

```diff
@@ -36,37 +36,118 @@ import classnames from 'classnames'
 /* eslint react/jsx-key: 0 */
 
 const propTypes = {
-  columns: PropTypes.arrayOf(Types.column).isRequired,
-  controlled: PropTypes.bool.isRequired,
-  data: PropTypes.arrayOf(PropTypes.object).isRequired,
-  disableSort: PropTypes.bool.isRequired,
+  children: PropTypes.node.isRequired,
+  className: PropTypes.string,
+  data: PropTypes.arrayOf(PropTypes.object),
+  initialAscending: PropTypes.bool,
+  initialColumn: PropTypes.string,
+  disableReverse: PropTypes.bool,
+  disableSort: PropTypes.bool,
+  controlled: PropTypes.bool,
+  onChange: PropTypes.func,
   rowComponent: Types.component,
   headerComponent: Types.component,
-  className: PropTypes.string,
-  ...sortablePropTypes,
 }
-
 const defaultProps = {
-  className: ''
+  className: '',
+  data: [],
+  initialAscending: true,
+  initialColumn: '',
+  disableReverse: false,
+  disableSort: false,
+  controlled: false,
+  onChange: noop,
 }
 
-function SortableTable ({
-  columns,
+function getInitialSortControls(initialColumn, columns) {
+  const defaultControls = {
+    initialSortPath: '',
+    initialSortFunc: null,
+    initialValueGetter: null,
+  }
+  if (!initialColumn) return defaultControls
```

Comment:
You can define this as a constant outside of the function.

Reply:
Done